src/formElementPlugins/fileUpload/localisation/default/strings.json

```diff
@@ -1,10 +1,12 @@
 {
-  "ERROR_FILE_ALREADY_UPLOADED": "File already uploaded",
-  "ERROR_FILE_TYPE_NOT_PERMITTED": "File type not permitted",
-  "ERROR_FILE_TOO_BIG": "File too big",
+  "ERROR_FILE_ALREADY_UPLOADED": "File already uploaded!",
+  "ERROR_FILE_TYPE_NOT_PERMITTED": "File type not permitted!",
+  "ERROR_FILE_TOO_BIG": "File too big!",
   "ERROR_TOO_MANY_FILES": "Too many files!",
-  "ERROR_UPLOAD_PROBLEM": "Problem uploading file",
+  "ERROR_UPLOAD_PROBLEM": "Problem uploading file!",
```

Comment:
I have a personal preference for no exclamation marks - but that is just me! :)

Reply:
LOL, yeah that's a good point actually. Not sure why I put them in. I can remove: b9ec31de